rust/src/sip/sip.rs

```diff
@@ -388,4 +569,22 @@ pub unsafe extern "C" fn rs_sip_register_parser() {
     } else {
         SCLogDebug!("Protocol detecter and parser disabled for SIP/UDP.");
```

Comment:
The only thing I'd change now I think is this line and the one further below.

```
Protocol detection and parsing disabled for UDP SIP
```
and likewise for the TCP.  Or just bring correct spelling to both. Easy to do with in a staging commit though.